softlearning/environments/adapters/robosuite_adapter.py

```diff
@@ -0,0 +1,144 @@
+"""Implements a RobosuiteAdapter that converts Robosuite envs into SoftlearningEnv."""
+
+from collections import OrderedDict
+
+import numpy as np
+import robosuite as suite
+import gym
+from gym import spaces
+
+from .softlearning_env import SoftlearningEnv
+from softlearning.environments.adapters.gym_adapter import GymAdapter
+
+ROBOSUITE_ENVIRONMENTS = {}
+
+def convert_robosuite_to_gym_obs_space(robosuite_obs_space):
+    assert isinstance(robosuite_obs_space, OrderedDict)
+    listDict = []
+    for key, value in robosuite_obs_space.items():
+        listDict.append((key, spaces.Box(
+            low=-float("inf"),
+            high=float("inf"),
+            shape=(len(value),),
+            dtype=np.float32)))
+    return spaces.Dict(OrderedDict(listDict))
+
+def convert_robosuite_to_gym_action_space(robosuite_action_space):
+    assert isinstance(robosuite_action_space, tuple)
+    return spaces.Box(
+        low=robosuite_action_space[0],
+        high=robosuite_action_space[1],
+        dtype=np.float32)
+
+class RobosuiteAdapter(SoftlearningEnv):
+    """Adapter that implements the SoftlearningEnv for Robosuite envs."""
+
+    def __init__(self,
+                 domain,
+                 task,
+                 *args,
+                 env=None,
+                 normalize=True,
+                 observation_keys=None,
+                 unwrap_time_limit=True,
```

Comment:
I don't think robosuite wraps their environments, so we can remove this argument.